packages/wasi-common/nodejs/exit.js

```diff
@@ -0,0 +1,4 @@
+export function exit(status) {
+  console.log(`[exit] Exit: ${JSON.stringify(status)}`);
+  process.exit(1);
```

Comment:
This should probably be code-specific:

```suggestion
  process.exit(status.tag === 'err' ? 1 : 0);
```